src/main/java/unrc/dose/User.java

```diff
@@ -142,4 +143,35 @@ private void load(final User u, final String name,
         u.set(ADMIN, admin);
     }
 
+	 /**
+     * this method remove logically a user.
+     * @param usernameUser this username is for delete logicaly,
+     * his account associate
+     * @param passwordUser this param is used for confirm the operation
+     * @return a boolean with this exit of remove or no remove the user
+     */
+	public static boolean deleteUser(final String usernameUser,
+		final String passwordUser) {
+		User user = User.findFirst("username = ? and password = ?",
+			usernameUser, passwordUser);
+        if (user != null) {
+			user.set("active_account", false);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+     * this method is for login the user.
+     * @param usernameUser this username is for login
+     * @param passwordUser this param is used for finally operation
+     * @return a boolean for see if coincide username and pass
```

Comment:
The description of the method should state something like `User's credential validation, given a particular user's credential, this method will validate if they are valid`

When documenting param and return tags one does not usually refer to the param one is documenting, for example `@param usernameUser this username is for login` should start with `username ...` but the param should be called `username` so it would look like this: `@param username user login identification`

The return tag should state something like `{@code true} iff the user's credential are valid and the user was successfully logged in` (*iff means if and only if*)